Patch104pZH/GameFilesEdited/Data/INI/Object/System.ini

```
@@ -395,7 +395,7 @@ Object GPSScrambler_InvisibleMarker
 
     DefaultConditionState
       Model = None
-      ParticleSysBone = None GPSMicrowaveScrambler
+      ParticleSysBone = None GPSMicrowaveScambler
```

Maybe could do with a comment to avoid future confusion?
```suggestion
      ParticleSysBone = None GPSMicrowaveScambler ; bone name is misspelled
```

---

It is an effect name. I think comment is not necessary for now.